examples/magento/getToken.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+use Denner\Client\MagentoClient;
+
+$config = require realpath(__DIR__ . '/../bootstrap.php');
+$params = array(
+);
+
+$client = MagentoClient::factory($config);
+
+$response = $client->integrationAdminToken(
+    array(
+        'username' => $config[MagentoClient::OPTION_USERNAME],
+        'password' => $config[MagentoClient::OPTION_PASSWORD],
+    )
+);
+
+
+//$i = 0;
+//$token = '';
+//
+//// Result is a plain string, not JSON as should be.
+//// Can't get resource (results in error for the malformed body), but can get each byte of the body
+//while (($c = $response->offsetGet($i++)) !== null) {
+//   $token .= $c;
+//}
+
+$token = $response->getResource()->get('response');
+
+var_dump('Token:', $token);
+
+$url = "http://" . $_SERVER['SERVER_NAME'] . dirname($_SERVER['REQUEST_URI']);
+
+$operations = array(
```

Comment:
What's the purpose of this?

Reply:
Speed for testing, prepares the other examples links to easily click-and-run them (using the Authorization token generated in this request)